features/enable_fips_container.feature

```diff
@@ -0,0 +1,97 @@
+
+@uses.config.contract_token
+Feature: FIPS enablement in lxd containers
+
+    @series.xenial
+    @series.bionic
+    @series.focal
+    @uses.config.machine_type.lxd.container
+    Scenario Outline: Attached enable of FIPS in an ubuntu lxd container
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token` with sudo
+        And I run `DEBIAN_FRONTEND=noninteractive apt-get install -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" -y openssh-client openssh-server strongswan openssl <libssl> libgcrypt20` with sudo, retrying exit [100]
+        And I run `ua enable fips<updates>` `with sudo` and stdin `y`
+        Then stdout matches regexp:
+            """
+            Warning: Enabling <fips-name> in a container.
+                     This will install the FIPS packages but not the kernel.
+                     This container must run on a host with <fips-name> enabled to be
+                     compliant.
+                     Please run `apt upgrade` to ensure all FIPS packages are updated to
```

Comment:
Maybe we should have this message after the operation is finished

Reply:
thats a good idea - I'll add it to post_enable if on a container

Reply:
I don't think it can be a "Notice" because we wouldn't know when to remove it